test/unit/controllers/consolidations/MucrOptionsControllerSpec.scala

```diff
@@ -101,6 +101,15 @@ class MucrOptionsControllerSpec extends ControllerLayerSpec with MockCache with
         status(result) mustBe BAD_REQUEST
         verify(page).apply(any(), any(), any())(any(), any())
       }
+
+      "a MUCR conforms with the regex but has been send but is over 35 characters long" in {
```

Comment:
Nit: This test name seems not being linguisticly correct. Although one should be able to tell what is it about.

Reply:
let me fix that after the current meeting :)

Reply:
This has been addressed on e0ed4ff7f88c32b3b67621212cefce3796005359